aerogear/aerogear-android-sdk
# ✨  add instructions on how to support self-signed certs. Update the

example app to make sure it can work with self signed certs

## Motivation

Make sure the example app can work with self-signed certs for local development and testing

## Description

* Add instructions on how to support self-signed certs
* Update the example app to load additional user added CAs
* Remove `ConnectionBuilderForTesting` as it is not safe to have it in the lib

## Progress

- [x] Done Task

## Additional Notes

To test:
* Provision Keycloak using mobile core. It should be running with https and use self-signed certs
* Follow the instructions in this PR to support the self-signed cert
* Update the configuration file of the example app to use the local keycloak instance
* Try user authentication from the example app. It should continue to work.
